### ISSUE TITLE
`--purs` flag no longer works in 0.8.0

### ISSUE THREAD

**Issue by joneshf** [natefaubion/purescript-psa#46]
# Reproduction
```Console
$ $(pwd)/purs --version
0.13.8
$ npx --package=purescript-psa@0.7.3 psa --purs=$(pwd)/purs
npx: installed 1 in 0.98s
           Src   Lib   All
Warnings   0     0     0  
Errors     0     0     0  
$ npx --package=purescript-psa@0.8.0 psa --purs=$(pwd)/purs
npx: installed 1 in 1.056s
/Users/joneshf/.npm/_npx/17686/lib/node_modules/purescript-psa/output/Effect.Exception/foreign.js:29
    throw e;
    ^

Error: `purs` executable not found.
    at Object.exports.error (/Users/joneshf/.npm/_npx/17686/lib/node_modules/purescript-psa/output/Effect.Exception/foreign.js:8:10)
    at Object.$$throw [as throw] (/Users/joneshf/.npm/_npx/17686/lib/node_modules/purescript-psa/output/Effect.Exception/index.js:41:45)
    at /Users/joneshf/.npm/_npx/17686/lib/node_modules/purescript-psa/output/Main/index.js:563:57
    at ChildProcess.<anonymous> (/Users/joneshf/.npm/_npx/17686/lib/node_modules/purescript-psa/output/Node.ChildProcess/foreign.js:125:17)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

---

It seems like it's related to calling out to `purs` directly instead of using the value passed in by the flag:

https://github.com/natefaubion/purescript-psa/blob/f21acc6ddabcfbfea0dcc8e505a11d879cf78506/src/Main.purs#L224

If that seems right, I can make a PR to fix that.

**Comment by natefaubion** [comment]
Oops! Yes, please!

**Comment by natefaubion** [comment]
This was published in 0.8.1

**Comment by joneshf** [comment]
Thanks!
